### ISSUE TITLE
Infinite Scroll in Menus

### ISSUE THREAD

**Issue by fieldchicken** [Tuxemon/Tuxemon#732]
I find it a bit tedious to spam the directional keys to move the cursor one space at a time to enter my character's name at the start of game. This is also an issue when buying multiple items from the mart. I think it would be great if the cursor would continue to move and the quantity of items would continue to increase as long as the user is holding down one of the directional keys.

**Comment by bitcraft** [comment]
Good point.  Lets nail down the parameters such as the delay for the key repeat, and the repeat speed.  any suggestions?

**Comment by Qiangong2** [comment]
Maybe about 3/4 a second?
